marktplaats/query.py

```diff
@@ -123,18 +123,17 @@ def __init__(
         self.body = self.response.text
         self.body_json = json.loads(self.body)
 
-        self._set_query_data()
-
-    def _set_query_data(self):
-        # more fields will be added
-        # for now, this is a nice way to get the total result count when looping through pages
-        self.total_result_count = self.body_json.get('totalResultCount')
-
-    def get_listings(self):
+    def get_listings(self) -> list[Listing]:
         listings = []
         for listing in self.body_json["listings"]:
             try:
-                listing_time = datetime.strptime(listing["date"], "%Y-%m-%dT%H:%M:%S%z")
+                date_str = listing["date"]
+                if date_str == "Eergisteren":
+                    listing_time = datetime.now() - timedelta(days=2)
+                elif date_str == "Gisteren":
+                    listing_time = datetime.now() - timedelta(days=1)
+                else:
+                    listing_time = datetime.strptime(date_str, "%d %b %y")
             except ValueError:
                 listing_time = None
```

Comment:
Could you add a logger warning similar to [this one](https://github.com/jensjeflensje/marktplaats-py/blob/b099edf049584f48f92a2782c71a4d5dff24510d/marktplaats/query.py#L145)? I'm hoping we can catch these errors sooner this way :)